tidy3d/web/api/webapi.py

```diff
@@ -926,6 +927,20 @@ def real_cost(task_id: str, verbose=True) -> float:
     return flex_unit
 
 
+@wait_for_connection
+def account(verbose=True) -> UserAccount:
+    account_info = UserAccount.get()
+    if verbose:
+        console = get_logging_console()
+        console.log(
+            f"Current FlexCredit balance: {account_info.credit} and Expiration date: {account_info.credit_expiration}. "
+            f"{account_info.allowance_cycle_type} FlexCredit balance: {account_info.allowance_current_cycle_amount} "
+            f"and Expiration date:{account_info.allowance_current_cycle_end_date}. "
+            f"Daily free Simulation counts: {account_info.daily_free_simulation_counts}."
```

Comment:
I think no need to capitalize Expiration and Simulation.

Does `daily_free_simulation_counts` give the daily allowance (e.g. 10) or how many remain for the current day, (e.g. 6 if a user has already run 4)?